nextflow.config

```diff
@@ -81,6 +149,9 @@ profiles {
         shifter.enabled        = false
         charliecloud.enabled   = false
     }
+    arm {
+        docker.runOptions = '-u $(id -u):$(id -g) --platform=linux/amd64'
```

Comment:
Good question. Since, I don't have access to an arm-based machine, I have not tested it yet. But given that arm environments support gpus, I guess that simply adding '--gpus all' to the docker.runOptions could work. Do you have any experience on this?

Reply:
Is there any example in another nf-core pipeline that requires gpus?

Reply:
Would it be ok to remove the arm profile for the moment and put in a future release, once we have tested it?